djconnectwise/sync.py

```diff
@@ -854,6 +854,9 @@ def _assign_field_data(self, instance, json_data):
             instance.inactive = json_data.get('inactive')
 
         instance.project_flag = json_data.get('projectFlag', False)
+        instance.timeEntryDiscussionFlag = json_data.get('timeEntryDiscussionFlag', False)
```

Comment:
These should be snake case (`time_entry_discussion_flag` instead of `timeEntryDiscussionFlag`) instead of camel case to keep consistency with our existing code and Python best practices.